lib/v2/screens/dashboard/interactor/mappers/transactions_state_mapper.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:seeds/v2/domain-shared/page_state.dart';
+import 'package:seeds/v2/screens/dashboard/interactor/viewmodels/transactions_list_state.dart';
+import 'package:async/async.dart';
+
+class TransactionsListStateMapper {
+  TransactionsListState mapResultToState(TransactionsListState currentState, Result transactionsResult) {
+    return transactionsResult.isError
```

Comment:
this is just personal opinion, but ternary operations dont always lead to more readable code.

Reply:
I agree that sometimes more concise code is not more readable...

Reply:
Often the simplest for (i... ) loops are much clearer than ... I guess the best example of this is reduce().... reduce is always very short, and also totally opaque